circleci/images/debbuilder/files/install-builddeps

```diff
@@ -23,17 +23,23 @@ echo "Writing /etc/apt/sources.list.d/pgdg.list..." >&2
 cat > /etc/apt/sources.list.d/pgdg.list <<EOF
 deb     http://deb.debian.org/debian ${codename}-backports                main
 deb-src http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg         main
-deb-src http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg-testing 12
+deb-src http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg-testing 13
 EOF
 
 echo "Running apt-get update..." >&2
 apt-get update
 
+# this line is to bypass password prompts
+echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 echo "Installing tools for building Debian packages..." >&2
 apt-get install -y --no-install-recommends \
+  apt-utils \
   devscripts \
   fakeroot \
-  lintian
+  lintian \
+  slapd \
+  ldap-utils
```

Comment:
I think I added these packages after getting this (found from history) :

```
Please enter the password for the admin entry in your LDAP directory.

Administrator password:
```

This would basically get stuck and after researching I resolved it with the installation of these packages. I haven't checked which package was having this but I think it was the case with pg12 as well so it might be that there is some change in the base image. ( I will verify this tomorrow)

Reply:
I think the error was resolved with https://github.com/citusdata/the-process/pull/38/files#diff-7bb2f9e010c3fa36b21914afb9ec77c4R33. I didn't get this error when I tried building without these two packages, `slapd`, `ldap-utils`.